test/kubemark/start-kubemark.sh

```diff
@@ -453,6 +511,16 @@ echo
 echo -e "Getting total hollow-nodes number:" >&2
 "${KUBECTL}" --kubeconfig="${KUBEMARK_KUBECONFIG}" get node | grep "hollow-node" | wc -l
 echo
+
+for (( rp_num=1; rp_num<=${SCALEOUT_RP_COUNT}; rp_num++ ))
+do
+  rp_kubeconfig="${RP_KUBECONFIG}-${rp_num}"
+  echo
+  echo -e "Getting total hollow-nodes number for RP-${rp_num}" >&2
+  "${KUBECTL}" --kubeconfig="${rp_kubeconfig}" get node | grep "hollow-node" | wc -l
+  echo
+done
+
```

Comment:
there are 4 "for" loop of rp_num  in main, these can be merge together, then logic for what is done to start kubemark RP will be clear.

Reply:
fixed.